engine/rowset/src/main/java/io/deephaven/engine/rowset/RowSet.java

```diff
@@ -116,15 +115,6 @@ default WritableRowSet invert(RowSet keys) {
      */
     WritableRowSet invert(RowSet keys, long maximumPosition);
 
-    /**
-     * For the given keys RowSet, under the assertion that none of them are present in the current RowSet, return the
-     * tentative insertion points in the current RowSet with the count for each of them
-     *
-     * @param keys the keys to identify insertion locations
-     * @return two TLongArrayLists; [0] contains the positions, [1] contains the counts.
-     */
-    TLongArrayList[] findMissing(RowSet keys);
```

Comment:
If this is the only place where the API dependency is, please update or remove the dependency in `engine/rowset/build.gradle`

Reply:
It looks like we might be able to get away with downgrading that from api to implementation, I'll look into that further.